spec.md

```diff
@@ -200,6 +200,15 @@ that contains both context and data).
   * If present, MUST adhere to the format specified in
     [RFC 3986](https://tools.ietf.org/html/rfc3986)
 
+### content-type
+* Type: String per [RFC 2046](https://tools.ietf.org/html/rfc2046)
+* Description: Describe the data encoding format 
+* Constraints:
+  * OPTIONAL
+  * If present, MUST adhere to the format specified in
+    [RFC 2046](https://tools.ietf.org/html/rfc2046)
+  * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
```

Comment:
Minor tweak: remove the space before the "*" so the media type example text is not under "Constraints".